src/ddprof_module_lib.cc

```diff
@@ -104,14 +105,16 @@ find_matching_segment(std::span<const Segment> segments,
 
 MatchResult find_match(std::span<const Mapping> executable_mappings,
                        std::span<const Segment> elf_load_segments) {
-  int nb_mappings = executable_mappings.size();
+  unsigned nb_mappings = executable_mappings.size();
```

Comment:
you should stick to int

Reply:
Done, though I am not sure I see the point considering sizes are unsigned.